packages/aws-cdk-lib/aws-stepfunctions-tasks/lib/evaluate-expression.ts

```diff
@@ -101,6 +101,7 @@ function createEvalFn(runtime: lambda.Runtime | undefined, scope: Construct) {
   const lambdaPurpose = 'Eval';
 
   const nodeJsGuids = {
+    [lambda.Runtime.NODEJS_20_X.name]: 'f66698ab-28ac-4879-83d1-615be69028bc',
```

Comment:
Curious where is this id from?

Reply:
confirmed with @colifran , these are random GUIDs, this one is generated using https://www.uuidgenerator.net/guid

Reply:
These are just uuids. The `SingletonFunction` construct uses this to help determine if a`Function` has already been created within the same scope. As a result, I think the only requirement is that the uuid assigned needs to be unique.